src/components-examples/material/button-toggle/index.ts

```diff
@@ -9,25 +9,31 @@ import {
 } from './button-toggle-exclusive/button-toggle-exclusive-example';
 import {ButtonToggleOverviewExample} from './button-toggle-overview/button-toggle-overview-example';
 import {ButtonToggleHarnessExample} from './button-toggle-harness/button-toggle-harness-example';
+import {ButtonToggleFormsExample} from './button-toggle-forms/button-toggle-forms-example';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
```

Comment:
We usually sort the `@angular` imports towards the top of the file.

Reply:
fixed